apps/api/src/app/integrations/usecases/consume-integration-store-telegram-mobile-link/consume-integration-store-telegram-mobile-link.usecase.ts

```
@@ -0,0 +1,170 @@
+import {
+  BadGatewayException,
+  ConflictException,
+  Injectable,
+  Logger,
```

Biome (2.4.15)

[error] Please use the PinoLogger from `@novu/application-generic` instead

(lint/style/noRestrictedImports)